testing/openblas/APKBUILD

```diff
@@ -10,53 +11,52 @@ license="BSD"
 depends=""
 depends_dev="gfortran"
 makedepends="$depends_dev perl linux-headers"
-install=""
 subpackages="$pkgname-dev $pkgname-doc"
-source="$pkgname-$pkgver.tar.gz::http://github.com/xianyi/OpenBLAS/archive/v$pkgver.tar.gz
+source="$pkgname-$pkgver.tar.gz::http://github.com/xianyi/$_pkgname/archive/v$pkgver.tar.gz
 	0001-Use-strsignal-instead-of-sys_siglist.patch
 	"
-
-_builddir="$srcdir"/OpenBLAS-$pkgver
-prepare() {
-	local i
-	cd "$_builddir"
-	for i in $source; do
-		case $i in
-		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
-		esac
-	done
-}
+builddir="$srcdir/$_pkgname-$pkgver"
 
 # DYNAMIC_ARCH: supported on x86* only, see https://github.com/xianyi/OpenBLAS/issues/709
 case "$CARCH" in
-x86 | x86_64) DYNAMIC_ARCH=1 ;;
-*) DYNAMIC_ARCH=0 ;;
+x86 | x86_64) _dynamic_arch=1 ;;
+*) _dynamic_arch=0 ;;
 esac
 
 # USE_OPENMP=0: see GCC bug 60035
 # NO_AFFINITY: don't expect to work on musl, also breaks R
 # NUM_THREADS=32: match CONFIG_NR_CPUS
-FLAGS="USE_OPENMP=0 NO_LAPACK=0 NUM_THREADS=32 MAJOR_VERSION=3 \
-	NO_AFFINITY=1 DYNAMIC_ARCH=$DYNAMIC_ARCH"
+_flags="USE_OPENMP=0
+	NO_LAPACK=0
+	NUM_THREADS=32
+	MAJOR_VERSION=3
+	NO_AFFINITY=1
+	DYNAMIC_ARCH=0
```

Comment:
Oh, thanks a lot for noticing it! This change isn’t intentional, I’ve accidentally commited wrong changes.